tests/deprecated/dts/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
+project(hello_world)
```

Comment:
Change this? Also the commit message for the last commit seems to need some fixing up.

```
This this covers to small aspects of DTS functionality, one that the
legacy generation script continues to generally function and that
FLASH_AREA id's are the same between the old and new generator.
```